platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/renderer/MapRenderer.java

```diff
@@ -30,7 +30,7 @@
 
   // Holds the pointer to the native peer after initialisation
   private long nativePtr = 0;
-
+  private boolean hasSurface = true;
```

Comment:
Can this field be `protected` instead of having a setter? It's always set from the extending object anyway.